blog-service/2024-10-18-manage.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Scan Budgets (Manage) 
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - scan-budgets
+  - manage
+hide_table_of_contents: true
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We are happy to introduce our new **Usage Management** tab under **Accounts** section, which allows you to define spending limits for queries to prevent unexpected charges. They help you to manage Sumo Logic credits, especially in pay-per-use scenarios, by setting limits on search volume.
```

Comment:
```suggestion
We are happy to introduce our new **Usage Management** tab under the **Accounts** section, which allows you to define spending limits for queries to prevent unexpected charges. They help you to manage Sumo Logic credits, especially in pay-per-use scenarios, by setting limits on search volume.
```